apps/frontend/src/app/pages/company/applications-job/applications-job.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="container mx-auto min-h-screen px-3">
+  <div class="pb-4 pt-6 flex justify-between flex-wrap gap-4">
+    <div class="flex gap-2 items-center cursor-pointer" [routerLink]="['/']">
+      <i class="bi bi-arrow-left text-base-100 text-xl"></i>
+      <p class="font-bold text-sm text-base-100 font-mono">Back</p>
+    </div>
+    <wagademy-input-search
+      placeholder="Search"
+      [value]="searchJob"
+      (valueChange)="searchJob = $event"
+    ></wagademy-input-search>
+    <div class="flex flex-wrap gap-2">
+      @for (filter of filters; track filters) {
+      <button
+        class="border border-base-200 hover:border-none font-mono rounded-full py-1 px-3 hover:bg-secondary"
+        [ngClass]="[
+          filter.name === selectedFilter
+            ? 'bg-secondary text-base-100 border-none'
+            : 'text-base-100 hover:text-base-100'
+        ]"
+        (click)="selectedFilter = filter.name"
+      >
+        {{ filter.name }}
+      </button>
+      }
+    </div>
+  </div>
+  <div class="flex gap-3 font-sans items-center pb-4">
+    <h2 class="font-medium text-base-100 text-4xl">Job title</h2>
+    <p class="text-sm text-base-100 font-bold">
+      | {{ jobs.length }} APPLICATIONS
+    </p>
+  </div>
+  <div
+    class="max-[767px]:justify-center flex-wrap justify-between flex gap-8 pt-6 xl:pt-0"
```

Comment:
Justify between works fine for look like the figma when we have 4 cards, but test with 2 or 3, it does not work good and for the user will be bad, maybe a justify-start should be an option